client/extensions/woocommerce/app/store-stats/store-stats-widget-list/index.js

```diff
@@ -74,111 +55,95 @@ class StoreStatsWidgetList extends Component {
 	};
 
 	render() {
-		const { siteId, statType, query, data, emptyMessage, widgets } = this.props;
-		const selectedIndex = this.getSelectedIndex( data.data );
-		const { loaded } = this.state;
-		const isLoading = ! loaded && ! ( data && data.data.length );
-		const hasEmptyData = loaded && data && data.data.length === 0;
-		let values = [];
-		let titles = null;
-		let widgetData = null;
-		let widgetList = null;
+		const { data, deltas, query, widgets } = this.props;
+		const selectedIndex = this.getSelectedIndex( data );
+		const firstRealKey = Object.keys( deltas[ selectedIndex ] ).filter( key => key !== 'period' )[ 0 ];
+		const sincePeriod = this.getDeltaByStat( firstRealKey );
+		const periodFormat = getPeriodFormat( query.unit, sincePeriod.reference_period );
+		const values = [
+			{
+				key: 'title',
+				label: translate( 'Stat' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **78** times:
`translate( 'Stats' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).